doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt

```diff
@@ -62,25 +63,26 @@ build==1.0.3
 cached-property==1.5.2
 cachetools==5.3.2
 catboost==1.2.2
-certifi @ file:///croot/certifi_1690232220950/work/certifi
-cffi==1.16.0
-charset-normalizer==3.3.1
+certifi==2023.11.17
+cffi @ file:///croot/cffi_1700254295673/work
```

Comment:
why is this changed to a local file?

Reply:
I'm not exactly sure why it's a local, but it came from the build doing pip freeze 😅 
This is the artifact file: https://ray-ci-artifact-branch-public.s3.amazonaws.com/248521af9ca0d68217365d3d3c9598a4142c912e/tmp/artifacts/.image-info/ray-ml:248521-py39-cpu_pip-freeze.txt